### PR TITLE
MNT Remove TODO comments

### PR DESCRIPTION
**Pull request by sabina-talipova**
## Parent issue
- https://github.com/silverstripe/.github/issues/116

### CODE REVIEW

File: src/Model/BlogPost.php [hunk 1]
```diff
@@ -334,7 +334,6 @@ public function getCMSFields()
                 ? $parent->Tags()
                 : BlogTag::get();
 
-            // @todo: Reimplement the sidebar
```
Comment (sabina-talipova) [inline src/Model/BlogPost.php:337]: Open ticket: https://github.com/silverstripe/silverstripe-blog/issues/729

Comment (ChloeSartorelli) [reply src/Model/BlogPost.php:337]: Please just remove all the code that's related to the sidebar instead.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: src/Model/BlogPost.php (hunk 1 above)
Comment (sabina-talipova) [reply src/Model/BlogPost.php:337]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**